src/client/Inode.h

```diff
@@ -198,6 +203,7 @@ struct Inode {
 
   list<Cond*>       waitfor_caps;
   list<Cond*>       waitfor_commit;
+  list<Cond*>	    waitfor_deleg;
```

Comment:
The problem is that we have a wait_on_list() function. Can that accept a vector? Might be good add a wait_on_vector, I guess?